_jdg-quickstarts

```diff
@@ -1 +1 @@
-Subproject commit 7be4b6dfe68cb0d5b623528af18d0bc39471552f
+Subproject commit db2f24eeb88d165544195f4a0696bb82a6a531b6
```

Comment:
I don't see a GA release for this so I created https://github.com/jboss-developer/jboss-jdg-quickstarts/issues/101